### ISSUE TITLE
SFC: `no-html-messages` not working

### ISSUE THREAD

**Issue by wolfgangwalther** [intlify/eslint-plugin-vue-i18n#108]
The following passes with `DONE  No lint errors found!`:

```vue
<i18n lang="yaml" locale="en">
html: "<i>test</i>"
</i18n>

<template>
  <div>
    {{ $t('html') }}
  </div>
</template>
```

I would expect it to throw an error for `no-html-messages`.

Looking at the tests I found this:

https://github.com/intlify/eslint-plugin-vue-i18n/blob/83cc9a37fcf1260a86821a1659ed78802e3b070f/tests/lib/rules/no-html-messages.ts#L80

I don't think the tests are currently testing correctly with this replace. I guess this replace has been added because of #107.

In fact, using the following in my SFC throws the correct error (`error: used HTML localization message (@intlify/vue-i18n/no-html-messages)`:

```vue
<i18n lang="yaml" locale="en">
html: "&lt;i>test&lt;/i>"
</i18n>

<template>
  <div>
    {{ $t('html') }}
  </div>
</template>
```

This is surely not what is to be expected?

**Comment by ota-meshi** [comment]
This issue is basically the same issue #107, so I will close it.

https://github.com/intlify/eslint-plugin-vue-i18n/issues/107#issuecomment-672409636
